doc/project/hw_dashboard.md

```diff
@@ -5,4 +5,8 @@ This dashboard gives the current status of
 designs within the OpenTitan project.
 See the [Hardware Development Stages]({{< relref "/doc/project/hw_stages.md" >}}) for description of the hardware stages and how they are determined.
 
+## ASIC Tool Flow Results
+- [Nightly DV regression results](https://reports.opentitan.org/hw/top_earlgrey/dv/summary.html)<br>
```

Comment:
Use two newlines for a break instead of HTML.

Reply:
This can probably be resolved. I did not find any HTML line breaks in changes to MarkDown files in this PR.

@gkelly I think you would need to approve the PR as you requested changes initially.